MekHQ/src/mekhq/campaign/finances/Money.java

```diff
@@ -102,6 +102,10 @@ public Money absolute() {
     }
 
     public Money plus(Money amount) {
+        if (amount == null) {
+            return plus(0L);
+        }
+
         return new Money(getWrapped().plus(amount.getWrapped()));
```

Comment:
Easy ternary candidate, plus why not instead return Money.zero() ?

Reply:
Preferred the simpler version.

Reply:
What do you mean simpler? Money.zero() is our normal way to do so and this can cause confusion in the future.

Reply:
`plus(Money.zero())` would just call this method again